pkgs/applications/editors/android-studio/default.nix

```diff
@@ -10,16 +10,16 @@ let
     inherit tiling_wm;
   };
   stableVersion = {
-    version = "2022.3.1.18"; # "Android Studio Giraffe (2022.3.1)"
-    sha256Hash = "sha256-JCFeEySmrJEYELLMGvstc1z3Rd+8BpGKQrjW+8a/dDM=";
+    version = "2022.3.1.19"; # "Android Studio Giraffe (2022.3.1)"
+    sha256Hash = "sha256-JQYl3KsYPgxo6/Eu+KUir3NpUn128e/HBPk8BbAv+p4=";
   };
   betaVersion = {
-    version = "2022.3.1.17"; # "Android Studio Giraffe (2022.3.1) RC 1"
-    sha256Hash = "sha256-Bpur9ytiwa0udiyYVxlfLT4M+ZcO5atQUQg/ForATP4=";
+    version = "2023.1.1.17"; # "Android Studio Hedgehog (2023.3.1)"
+    sha256Hash = "sha256-0sN+B1RxxlbgxXrEs8gft4qjvIYtazJS6DllHZ2h768=";
   };
   latestVersion = {
-    version = "2023.1.1.14"; # Android Studio Hedgehog (2023.1.1) Canary 14
-    sha256Hash = "sha256-MgZDUpRyM0XDfgqfYjtJJyG2CBaNB06PgtoJltDwNwk=";
+    version = "2023.2.1.1"; # Android Studio Iguana (2023.2.1) Canary 1
+    sha256Hash = "sha256-7ub9GkJd9J37nG2drXOuU7r4w/gI+m9nlWANqIk2ddk=";
```

Comment:
Seems like there is a mistake with the  Canary version in the comment shouldn't it be 15?

Reply:
Not as far as I understand: `Giraffe` went to stable, `Hedgehog` to beta and `Iguana` became canary.

With `Iguana` the minor was incremented and patch was reset to 1:

![image](https://github.com/NixOS/nixpkgs/assets/7900887/1454c77d-cca6-471d-8e47-2eb2ef7fc439)

Source: https://archive.ph/BNZik

Reply:
Okay fair enough